src/DevChatter.DevStreams.Infra.Db/Migrations/AddChannelImageToTwitchChannel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using FluentMigrator;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DevChatter.DevStreams.Infra.Db.Migrations
+{
+    [Migration(201904182000)]
+    public class AddChannelImageToTwitchChannel : Migration
+    {
+        private const string ColumnName = "ImageUrl";
+        private const string TableName = "TwitchChannels";
+
+        public override void Up()
+        {
+            Alter.Table(TableName)
+                .AddColumn(ColumnName)
+                .AsString(4000) // Note: 4000 is Db max value in Fluent Nhibernate. Not sure if this is a good idea or not? -- DAR 18/04/2019
```

Comment:
We're not using Fluent NHibernate, so I'd ignore its limits. If we transition to it, we could likely handle changing it.

Reply:
Okay, that sounds good to me. 
Sorry, I should've been clearer. It was more of a 'This works for Fluent Nhibernate, will it work for Fluent migrator?' sort of thing. 

Are you happy with the 4000 limit here?